doc/users/prev_whats_new/changelog.rst

```diff
@@ -4879,7 +4879,7 @@ the `API changes <../../api/api_changes.html>`_.
 2005-05-12
     Started work on TeX text for antigrain using pngdvi -- see
     examples/tex_demo.py and the new module matplotlib.texmanager.  Rotated
-    text not supported and rendering small glyps is not working right yet.  BUt
+    text not supported and rendering small glyphs is not working right yet.  BUt
```

Comment:
```suggestion
    text not supported and rendering small glyphs is not working right yet.  But
```

Reply:
In case people are wondering why the tool I'm using didn't trip on this, it parses it as `B`+`Ut` and I'm pretty sure it's in the dictionary I'm using:

> [Definition of ut](https://www.google.com/url?sa=t&source=web&rct=j&url=https://www.merriam-webster.com/dictionary/ut%23:~:text%3DDefinition%2520of%2520ut,and%2520later%2520replaced%2520by%2520do&ved=2ahUKEwi5tp7j8MX3AhWiGDQIHWeLCGsQFnoECA4QBQ&usg=AOvVaw1UyOtShE-0TyWSPn7HAFT5)
(Entry 1 of 2) : a syllable used for the first note in the diatonic scale in an early solmization system and later replaced by do.

Eventually I reached the conclusion that two character "words" weren't worth spell-checking. It's tunable, so a project could. But I'd be more likely to add a forbidden rule for the specific example if it seems like it's a common error. I haven't noticed it much, but I also haven't looked for it.